app/controllers/api/person_escort_records_controller.rb

```diff
@@ -17,6 +22,14 @@ def create
       render_person_escort_record(person_escort_record, :created)
     end
 
+    def update
+      PersonEscortRecords::ParamsValidator.new(update_person_escort_record_attributes).validate!
+      person_escort_record = PersonEscortRecord.find(params[:id])
```

Comment:
Might be nice to add a `person_escort_record` method as this is now used in a couple of places.

```ruby
def person_escort_record
  @person_escort_record ||= PersonEscortRecord.find(params[:id])
end
```